Logger/LoggerTask/Logger.cs

```diff
@@ -0,0 +1,130 @@
+﻿using ListenersConfigurationLibrary.Enums;
+using ListenersConfigurationLibrary.Interfaces;
+using ListenersConfigurationLibrary.ListenerConfigurations;
+using ListenersConfigurationLibrary.ListenersConfigurations;
+//using ListenersLibrary.Listeners;
+using Microsoft.Extensions.Configuration;
+using System.Reflection;
+using System.Text;
+
+namespace LoggerTask
+{
+    public class Logger
+    {
+        public List<IListener> _listeners = new List<IListener>();
+        private string[] _paths = { "ListenersLibrary.FileListener", "ListenersLibrary.WordListener", "ListenersLibrary.EventLogListener" };
+        private string _propertyAttributeName;
+        private string _propertyValue;
+        private string _fieldAttributeName;
+        private string _fieldValue;
+        private string _fileName;
+        private string _minLogLevel;
+        private string _eventLogName;
+
+        public Logger()
+        {
+            IConfigurationBuilder builder = new ConfigurationBuilder()
+               .AddJsonFile(@"C:/Users/AlgirdasCernevicius/source/repos/dotNET.Training/Logger/LoggerTask/appsettings.json");
+            var config = builder.Build();
+            foreach (var item in _paths)
+            {
+                switch (item)
+                {
```

Comment:
Bad idea

Reply:
Listner1.dll
Listner2.dll
Listner 3.dll

Reply:
IListner

Reply:
Separate listeners in differenr assemblies, use only the name of dll from apsettings